docs/fieldnorm/life_cycle.md

```diff
@@ -1,7 +1,56 @@
+
+  c.       Write code with unit and integration tests. Moreover, conducting regular code reviews is highly recommended. These steps are crucial for code verification and will be invaluable during software maintenance, testing, and release phases. It enables a wider understanding of parts of the code as unit tests often embody general usage as well as very specific exceptions and code reviews enforce colleagues to have a look and understand what your code is doing.
+
+  d.       Establish a well-defined development and release cycle with set time periods for preparation, development, testing & releasing. This allows multiple developers to work cohesively on the project and creates structured testing and release moments during the implementation and maintenance processes. It also enables ample opportunities for user reflections on the deliverables during the lifetime of project.
+
+5.      Testing
+Before implementation, software should be extensively tested, verified and validated. While ISO 15189 does not provide specific guidelines on how to conduct software validation, it emphasizes the importance of ensuring that software is validated and verified following recognized industry standards. We advise the use of the software validation and verification procedures as described in this guideline.
+
+6.      Implementation
+Once everything is ready, a clear time point should be determined to release the software. The exact time point at which a software is released can be determined by the Development & Release cycle or released in an independent matter. The software functionality and release date should be clearly communicated with the end users.
+
+7.      Maintenance
+Maintenance is primarily centers around fixing bugs, monitoring the software, introducing new features and upgrading to newer versions with new end-users requirements . There should be a clear guideline for how new features, bug fixes and new versions are implemented into the software, preferably using the Development & Release cycle. A systematic approach will streamline the maintenance process and improve overall software quality and reliability.
+
+Development & Release cycle
+Bioinformaticians are advised to have a clearly determined Development & Release cycle procedure with set time periods for preparation, development and testing & releasing of features. Having a Development & Release cycle allows multiple developers to work on the same project in an organized manner. It also creates a platform for creating structural testing and release moments for the implementation and maintenance SDLC process steps. An example of a release cycle would be 1 week of preparation, 3 weeks of development followed by 1 week of Testing & releasing which overlaps with the first week (preparation) of the next cycle, resulting in a development & release cycle of 4 weeks.
```

Comment:
this would be fantastic, but I think most of the groups don't do the release cycle in such a detailed way. 
I would leave this with 'how to do a release' 
In Utrecht we develop,and when we finish all the features in a specific release, we do verification, validation and release (and update all docs). This is not a set cycle.